server/app/views/applicant/ApplicantProgramFileUploadBlockEditTemplate.html

```diff
@@ -0,0 +1,115 @@
+<!doctype html>
+<html>
+  <head th:replace="~{applicant/ApplicantBaseFragment :: pageHeader}"></head>
+  <!-- TODO(#6902): This data attribute is waited for in screenshot tests. Update once we have modal dialogs -->
+  <body data-load-modal="true">
+    <div
+      th:with="question=${applicationParams.block().getQuestions().get(0)},
+               questionParams=${questionRendererParams.get(question.getQuestionDefinition().getId())},
+               fileUploadQuestion=${question.createFileUploadQuestion()}"
+    >
+      <form
+        enctype="multipart/form-data"
+        method="POST"
+        id="cf-block-form"
+        th:class="${fileUploadViewStrategy.getUploadFormClass()}"
+        th:action="${fileUploadViewStrategy.formAction(questionParams.signedFileUploadRequest().get())}"
+      >
+        <div
+          th:replace="~{applicant/ApplicantBaseFragment :: progressBar}"
+        ></div>
+
+        <div
+          th:replace="~{applicant/ApplicantBaseFragment :: requiredFieldsExplanation}"
+        ></div>
+
+        <div
+          th:replace="~{questiontypes/FileUploadQuestionFragment :: file-upload(${question}, ${questionParams}, ${fileUploadViewStrategy})}"
+        ></div>
+      </form>
+
+      <!-- Hidden forms that allow users to pass through without submitting a file.  -->
+      <div class="hidden">
+        <form
+          id="cf-fileupload-continue-form"
+          method="POST"
+          th:action="${submitFormAction}"
+        >
+          <input hidden th:value="${csrfToken}" name="csrfToken" />
+          <div
+            th:replace="~{questiontypes/FileUploadQuestionFragment :: file-key-field(${fileUploadQuestion}, false)}"
+          ></div>
+        </form>
+        <form
+          id="cf-fileupload-delete-form"
+          method="POST"
+          th:action="${submitFormAction}"
+        >
+          <input hidden th:value="${csrfToken}" name="csrfToken" />
+          <div
+            th:replace="~{questiontypes/FileUploadQuestionFragment :: file-key-field(${fileUploadQuestion}, true)}"
+          ></div>
+        </form>
+      </div>
+
+      <div
+        th:with="hasUploadedFile=${fileUploadQuestion.getFileKeyValue().isPresent()}"
+      >
+        <!-- Review page button -->
+        <button
```

Comment:
**thought:** Since this is similar to the next one you could consider creating a fragment for it with the name as an input, but I'm fine leaving it as is too. Same with some of the other buttons with overlap.

Reply:
done